src/components/ec-radio-btn-group/ec-radio-btn-group.spec.ts

```diff
@@ -0,0 +1,212 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtn, EcRadioBtnGroup } from '../../main';
+import type { RadioButtonGroupProps } from './types';
+
+function mountRadioBtnGroup(props?: Partial<RadioButtonGroupProps>, mountOpts?: MountingOptions<RadioButtonGroupProps>) {
+  return mount(
+    EcRadioBtnGroup as any, // eslint-disable-line
+    {
+      props: {
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        name: 'testingNameProp',
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtnGroup();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    it('should render the child component (radio-btn) with all the given props', () => {
+      const wrapper = mountRadioBtnGroup({
+        options: [{ value: 'y', label: 'Yes', description: 'Confirm' }, { value: 'n', label: 'No', description: 'Reject' }],
+        isDisabled: true,
+        isTextInline: true,
+        modelValue: 'y',
+        errorMessage: 'A testing error message',
+      });
+      const givenPropsRadio1 = wrapper.findAllComponents(EcRadioBtn)[0].vm;
+      const givenPropsRadio2 = wrapper.findAllComponents(EcRadioBtn)[1].vm;
+
+      const expectedPropsRadio1 = {
+        isDisabled: true,
```

Comment:
we need to check also that `name` prop of these components props are the same

Reply:
isn't it already tested on line 77?

Reply:
https://github.com/Ebury/chameleon/pull/1386#discussion_r1048561782